runtime/queries/org/highlights.scm

```diff
@@ -0,0 +1,24 @@
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "*")) @markup.heading.1
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "**")) @markup.heading.2
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "***")) @markup.heading.3
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "****")) @markup.heading.4
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "*****")) @markup.heading.5
+(headline (stars) @markup.heading.marker (#eq? @markup.heading.marker "******")) @markup.heading.6
+
+ (block) @markup.raw.block
+ (list) @markup.list.unnumbered
+ (directive) @markup.label
+ (property_drawer) @markup.label
+ 
+
+ ((expr) @markup.bold
+  (#match? @markup.bold "\\*.*\\*"))
+ 
+ ((expr) @markup.italic
+  (#match? @markup.italic "/.*/"))
+
+ ((expr) @markup.raw.inline
+  (#match? @markup.raw.inline "~.*~"))
+ 
+ ((expr) @markup.quote
+  (#match? @markup.quote "=.*="))
```

Comment:
```suggestion
(block) @markup.raw.block
(list) @markup.list.unnumbered
(directive) @markup.label
(property_drawer) @markup.label
 

((expr) @markup.bold
 (#match? @markup.bold "\\*.*\\*"))
 
((expr) @markup.italic
 (#match? @markup.italic "/.*/"))

((expr) @markup.raw.inline
 (#match? @markup.raw.inline "~.*~"))
 
((expr) @markup.quote
 (#match? @markup.quote "=.*="))

```

looks like some extra spaces snuck in here, and there's a missing trailing newline. Would you mind adding one to the end of `injections.scm` as well?

Reply:
Did I ever tell you I suck at Git? Cause I do, I think now the spaces are gone (atleast I think so), and newlines are added, let me know if anything else needs fixing.